File: wfe-spa/src/ts/WfReportDto.ts
export class WfReportDto {
    id: number|null = null;
    name: string = '';
    description: string = '';
    category: string = '';
    parameters: Array<WfReportParameterDto>|null = null;

    getInfo(): object {
        return {
            'Название': this.name,
            'Описание': this.description
        };
    }
}
class WfReportParameterDto{
	userName: string = '';
    description: string = '';
    internalName: string = '';
    position: number|null = null;
    type: any;
    required: boolean|null = null;
    value: any = '';
}

File: wfe-restapi/src/main/java/ru/runa/wfe/rest/dto/WfReportDto.java
package ru.runa.wfe.rest.dto;

import java.util.ArrayList;
import java.util.List;

import lombok.Data;

@Data
public class WfReportDto {
    private Long id;
    private Long version;
    private String name;
    private String description;
    private String category;
    private List<WfReportParameterDto> parameters = new ArrayList<>();
}
